client/client.go

```diff
@@ -94,16 +94,24 @@ func ValidRedirectURL(rURL *url.URL, redirectURLs []url.URL) (url.URL, error) {
 	return url.URL{}, ErrorInvalidRedirectURL
 }
 
-func ClientsFromReader(r io.Reader) ([]Client, error) {
+// LoadableClient contains sufficient information for creating a Client and its related entities.
+type LoadableClient struct {
```

Comment:
It's also used in db/client.go.

I see what you're saying about the weirdness, but my rationale is this:

1) this is the same model that the user module uses. There's something to be said for consistency.

2) when we add more storage backends (as we will almost certainly do it seems) they will also need something in their implementation which loads such "full" clients.